crates/frontend/src/components/context_form.rs

```diff
@@ -24,6 +25,7 @@ pub fn condition_input(
     #[prop(into)] on_remove: Callback<String, ()>,
     #[prop(into)] on_value_change: Callback<Expression, ()>,
     #[prop(into)] on_operator_change: Callback<Operator, ()>,
+    tooltip_text: String,
```

Comment:
Should there be a default prop on tooltip_text? Makes it easier to ignore if you don't need it

```suggestion
    #[prop(default = String::new())]  tooltip_text: String,
```